src/System.Security.Cryptography.Cng/tests/ImportExportTests.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Cng.Tests
+{
+    public static class ImportExportTests
+    {
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public static void TestImportExport1()
+        {
+
+            byte[] reExported;
```

Comment:
Probably inheriting the pattern of other (private) tests where the "using" line actually creates a key on disk, so I want to get rid of it as fast as I can because I get tired of cleaning up turd off my machine.

Not a factor here since I'm only checking in non-invasive tests. I'll put this on the "clean up if I rev the PR again" list.